templates/compositions/ec-component-organisation-teaser/organisation-teaser.html.twig

```diff
@@ -0,0 +1,119 @@
+{% spaceless %}
+
+{#
+/**
+ * @file
+ * Theme implementation for ECL Organisation Teaser.
+ *
+ * Available variables:
+ * - "name": name of the organisation.
+ * - "acronym": acronym of the organisation.
+ * - "logo": logo of the organisation.
+ * - "organisation_details": list of fields.
+ * - "contact_page_url": email adress or contact page to contact the organisation.
+ * - "contact_page": array with processed organisation contact page url.
+ */
+#}
```

Comment:
We could add the type and default value for the variables, check this as an example: https://github.com/openeuropa/oe_theme/blob/2.x/templates/compositions/ec-component-navigation-list/navigation-list.html.twig

Reply:
Done